File: C/bee 1060.c
#include <stdio.h>
#include <math.h>

int main(void) {

    int qtdPositivos = 0;
    for (int i = 0; i < 6; i++) {
        double n;
        scanf("%lf", &n);
        if (n > 0) {
            qtdPositivos++;
        }
    }

    printf("%d valores positivos\n", qtdPositivos);

    return 0;
}

File: Rust/bee_1060.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    
    let valores: Vec<f64> = input
        .split_whitespace()
        .map(|x| x.parse().unwrap())
        .collect();

    let mut qtd_positivos = 0;
    for i in 0..6 {
        if valores[i] > 0.0 {
            qtd_positivos += 1;
        }
    }

    println!("{} valores positivos", qtd_positivos);
}
